client/rhel/rhn-client-tools/rhn-client-tools.spec

```diff
@@ -256,11 +270,17 @@ Requires: rhn-setup-gnome = %{version}-%{release}
 Requires: python-gnome python-gtk
 %else
 Requires: pygtk2 pygtk2-libglade
+%if 0%{?mageia}
+Requires: usermode
+%else
```

Comment:
This %if %else condition is missing the %endif section which causes the travis build to fail.

Reply:
Should be fixed now.